drivers/unifiedPlatform/pds/backend/v1/api/deploymentsConfigUpdate.go

```diff
@@ -29,7 +31,7 @@ func (ds *PDS_API_V1) UpdateDeployment(updateDeploymentRequest *automationModels
 		return nil, fmt.Errorf("Error when calling `UpdateDeployment`: %v\n.Full HTTP response: %v", err, res)
 	}
 
-	err = copier.Copy(&dsResponse, dsModel)
+	err = utilities.CopyStruct(dsModel, dsResponse.Update)
```

Comment:
Please use &dsResponse.Update instead of dsResponse.Update as it should be pass by reference. Please change this in all places if direct variable is used.

Reply:
done